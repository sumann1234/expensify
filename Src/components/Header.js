import React from 'react';
import {NavLink} from 'react-router-dom';

const Header=() =>(
    <header>
    <h1>Expensifhhhjy</h1>
    <NavLink to="/" activeClassName='active-class' exact={true}> Home</NavLink>
    <NavLink to="/create" activeClassName='active-class'> Add </NavLink>
   
    <NavLink to="/help" activeClassName='active-class'> Help </NavLink>
    </header>
)

export default Header;