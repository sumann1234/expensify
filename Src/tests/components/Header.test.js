import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import {shallow} from 'enzyme';
import Header from '../../components/Header';
import toJSON from 'enzyme-to-json';


test('testing the Header',() =>{
    const wrapper= shallow(<Header/>);
    expect((wrapper)).toMatchSnapshot();
    
    // const renderer = new ReactShallowRenderer;
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot()
})