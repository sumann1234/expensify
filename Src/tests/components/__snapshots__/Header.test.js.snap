// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing the Header 1`] = `
<header>
  <h1>
    Expensifhhhjy
  </h1>
  <[object Object]
    activeClassName="active-class"
    exact={true}
    to="/"
  >
     Home
  </[object Object]>
  <[object Object]
    activeClassName="active-class"
    to="/create"
  >
     Add 
  </[object Object]>
  <[object Object]
    activeClassName="active-class"
    to="/help"
  >
     Help 
  </[object Object]>
</header>
`;
